lib/minitest_spec_context.rb

```diff
@@ -0,0 +1,9 @@
+require 'minitest/spec'
+
+module MiniTest
```

Comment:
Please keep this in `test/test_helper.rb` or some file in that directory. It looks strange having it in `lib` which should not contain test-related code